src/Aspire.Hosting/ApplicationModel/IInteractionService.cs

```diff
@@ -321,6 +321,12 @@ public class InteractionOptions
     /// Gets or sets a value indicating whether show the dismiss button in the header.
     /// </summary>
     public bool? ShowDismiss { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the message is rendered as Markdown.
+    /// Setting this to <c>true</c> allows a message to contain Markdown elements such as links, text decoration and lists.
+    /// </summary>
+    public bool? MessageAsMarkdown { get; set; }
```

Comment:
Suggestions for a better name?

Reply:
I think `SupportMarkdownMessage` is what I'll go with.

Reply:
Leave it as is. It's preview API. We can tweak later.

Reply:
lol I see you changed it. Looks good.